pavics_datacatalog/wps_processes/wps_pavicsearch.py

```diff
@@ -174,47 +190,74 @@ def _handler(self, request, response):
         # Not sure if the default should actually be forced to None here...
         fields = request.inputs['fields'][0].data
         if 'constraints' in request.inputs:
+            logger.info('using constraints : {0}'.format(request.inputs['constraints'][0].data))
             constraints = request.inputs['constraints'][0].data
         else:
             constraints = None
         if 'query' in request.inputs:
+            logger.info('using query : {0}'.format(request.inputs['query'][0].data))
             query = request.inputs['query'][0].data
         else:
             query = None
         if 'esgf' in request.inputs:
+            logger.info('using esgf : {0}'.format(request.inputs['esgf'][0].data))
             search_esgf = request.inputs['esgf'][0].data
         else:
             search_esgf = False
         if 'list_type' in request.inputs:
+            logger.info('using list_type : {0}'.format(request.inputs['list_type'][0].data))
             list_type = request.inputs['list_type'][0].data
         else:
             list_type = 'opendap_url'
 
         if search_esgf:
             try:
+                logger.info('calling pavics_and_esgf_search to {host} and esgf host {esgf} with : '
+                            'facets={facets}, offset={offset}, limit={limit}, '
+                            'fields={fields}, query={query}, constraints={constraints}, '
+                            'search_type={search_type}, output_format={output_format}'.format(
+                    host=self.solr_server, esgf=str(self.esgf_nodes),
+                    facets=facets, offset=offset,
+                    limit=limit, fields=fields, query=query,
+                    constraints=constraints, search_type=search_type,
+                    output_format=output_format
+                ))
                 search_result = catalog.pavics_and_esgf_search(
                     [self.solr_server], self.esgf_nodes, facets=facets,
                     offset=offset, limit=limit, fields=fields, query=query,
                     constraints=constraints, search_type=search_type,
                     output_format=output_format)
             except:
+                logger.error(traceback.format_exc())
                 raise Exception(traceback.format_exc())
         else:
             try:
+                logger.info('calling pavicsearch to {host} with : '
+                            'facets={facets}, offset={offset}, limit={limit}, '
+                            'fields={fields}, query={query}, constraints={constraints}, '
+                            'search_type={search_type}, output_format={output_format}'.format(
+                    host=self.solr_server, facets=facets, offset=offset,
+                    limit=limit, fields=fields, query=query,
+                    constraints=constraints, search_type=search_type,
+                    output_format=output_format
+                ))
```

Comment:
to remove big lists like so (top & bottom), we could consider writing to a 'params' dict and simply unpack with `**params` ?